sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -375,7 +378,7 @@ public void setExecutorVolume(VolumeSpec volumeSpec) {
      */
     @VisibleForTesting
     public static Map<String, String> getTaskEnvironment(
-            String serviceName, PodInstance podInstance, TaskSpec taskSpec) {
+            String serviceName, PodInstance podInstance, TaskSpec taskSpec, SchedulerConfig schedulerConfig) {
```

Comment:
In terms of the hierarchy in the `svc.yml` file, we have:
* service (`name`)
* scheduler
* pods
* tasks

Would it make sense to move the `schedulerConfig` parameter to after `serviceName` to better reflect this?

Reply:
SchedulerConfig is the class for accessing the environment of the scheduler. It isn't a part of the yaml.

Reply:
Ok. The name was a bit confusing. I suppose `SchedulerMarathonEnv` would be more accurate. (not that it should change now).